### PR TITLE
Feature: adding grpc login client e2e test

### PR DESCRIPTION
**Pull request by eapg**
* Adding Login Client credential e2e test.

### CODE REVIEW

File: src/test/java/com/ea/restaurant/grpc/GrpcOauth2ServiceImplE2eTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import net.devh.boot.grpc.server.autoconfigure.GrpcServerFactoryAutoConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@SpringBootTest(
```
Comment (reynoldmorel) [inline src/test/java/com/ea/restaurant/grpc/GrpcOauth2ServiceImplE2eTest.java:22]: Create a properties same ss you did on your code base

File: src/test/resources/application.properties [hunk 2]
```diff
@@ -2,3 +2,5 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/ea_restaurant_java_etl_test
 spring.datasource.username=postgres
 spring.datasource.password=1234
+
+oauth2.secret.key=thisIsTheSecretKeyForJava-Etlgrp
```
Comment (reynoldmorel) [inline src/test/resources/application.properties:6]: Add new line

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/test/resources/application.properties (hunk 2 above)
Comment (eapg) [reply src/test/resources/application.properties:6]: Done.

File: src/test/java/com/ea/restaurant/grpc/GrpcOauth2ServiceImplE2eTest.java (hunk 1 above)
Comment (eapg) [reply src/test/java/com/ea/restaurant/grpc/GrpcOauth2ServiceImplE2eTest.java]: done.

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**